src/hooks/useHandleNFTRedeem.tsx

```diff
@@ -197,22 +210,64 @@ export function useHandleNFTRedeem() {
   const signedVAA = useNFTSignedVAA();
   const isRedeeming = useSelector(selectNFTIsRedeeming);
   const handleRedeemClick = useCallback(() => {
+    const telemetryProps: TelemetryTxEvent = {
+      fromChainId: undefined,
```

Comment:
You could obtain the fromChainId with this`const targetChain = useSelector(selectAttestTargetChain);`